docs/pages/getting-started/6-adding-authorizations.rst

```diff
@@ -17,14 +17,22 @@ An authorization credential is a privately distributed credential that answers:
 Authorization credentials are issued by the same party that will also control the access to the resources.
 
 Bolt
-****
+====
```

Comment:
ReST does not have hardcoded meanings for the headers. The first time it finds a new one, it is regarded as a subheader of the one above it.
This means that the paragraph `Registering a NutsAuthorizationCredential` is now a sub-paragraph of `Prerequisites`?

Reply:
Inserted an "Introduction" heading to fix it